css/css-syntax/escaped-eof.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<title>Escaped EOF</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<meta name="author" title="Tab Atkins-Bittner">
+<link rel=help href="https://drafts.csswg.org/css-syntax/#consume-escaped-code-point">
+<link rel=help href="https://drafts.csswg.org/css-syntax/#consume-string-token">
+<link rel=help href="https://drafts.csswg.org/css-syntax/#consume-token">
+
+<style>foo { --foo:foo\</style>
+<style>foo { --foo:1foo\</style>
+<style>foo { --foo:url(foo\</style>
+<style>foo { --foo:"foo\</style>
+
+<script>
+
+function threw(fn) {
+    try{fn();}
+    catch(e) { return true; }
+    return false;
+}
+
+test(()=>{
+    assert_true(threw(()=>{document.querySelector("#123");}), "numeric hash token is invalid in a selector");
```

Comment:
Use assert_throws, so you can test the exception type too?

Reply:
Was using it originally, but didn't like the asymmetry in my second condition. That said... (see next response).